### PR TITLE
[HER-51] Create business repository

### PR DESCRIPTION
**Pull request by andreea-dumitrascu**
## Description
- Create detection repository interface
- It should live inside business layer
- It shouldn't be referenced anywhere yet
- It should contain a single method
- All -> IEnumerable<MDetection>

## Related
[HER-51](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?selectedIssue=HER-51)

### CODE REVIEW

File: src/Business/Detection/Repositories/IDetection.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+namespace Business.Detection.Repositories
+{
+    using System.Collections.Generic;
+    using Business.Detection.Models;
+
+    public interface IDetection
```
Comment (RaduBuciuceanu) [inline src/Business/Detection/Repositories/IDetection.cs:6]: Would the name: `IDetectionRepository` be more relevant here?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Business/Detection/Repositories/IDetection.cs (hunk 1 above)
Comment (andreea-dumitrascu) [reply src/Business/Detection/Repositories/IDetection.cs]: Done!

**Review by mihairadulescu — DISMISSED** [review]
rebase and merge

**Review by miutamihai — APPROVED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]
Rebase, rebase, rebase!

**Comment by andreea-dumitrascu** [comment]
> Rebase, rebase, rebase!

Done!

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**